README.md

```diff
@@ -84,12 +84,14 @@ The mixin comes with some options. Due to the way listeners and `IntersectionObs
 
 ```js
 import Component from '@ember/component';
+import { setProperties } from '@ember/object';
 
 export default Component.extend(InViewportMixin, {
   init() {
     this._super(...arguments);
 
-    Ember.setProperties(this, {
+    setProperties(this, {
+      viewportDescriptor              : null,
```

Comment:
Not sure I like adding a new API item for this.  Another potential option is to compare the options (see PR notes) and make the `key` in the "cache" as some unique identifier object.

https://github.com/DockYard/ember-in-viewport/pull/156/files#diff-554ac2552b81af2c27b0b894b55d1009R4